frontend/packages/kubevirt-plugin/package.json

```diff
@@ -7,7 +7,9 @@
     "@console/internal": "0.0.0-fixed",
     "@console/plugin-sdk": "0.0.0-fixed",
     "@console/shared": "0.0.0-fixed",
-    "kubevirt-web-ui-components": "^0.1.34-4"
+    "kubevirt-web-ui-components": "^0.1.34-4",
+    "kubevirt-typescript-api": "^0.18.1-rc",
+    "openshift-typescript-api": "^4.1.0-rc1"
```

Comment:
added all TS definitions of openshift/kubevirt objects. These are generated from openapi specifications with few manual changes and released as `kubevirt-typescript-api` and `openshift-typescript-api` packages.

Reply:
maybe it would make sense to move this into main `package.json` and use all the types across the whole project

Reply:
thoughts?